.github/workflows/ecosystem-recompile.yml

```diff
@@ -0,0 +1,43 @@
+name: Ecosystem | Recompile members.json
+
+on:
+  push:
```

Comment:
Hmm maybe not at push ? Could be too much ? (Not sure)
But we could scheduled it to have it like every week or day :
```
  schedule:
    - cron:  '0 3 * * 0'
 ```

It's just an example : every week the Sunday at 3am

Reply:
I think push should be OK as we don't change the members that often, and this action should only push commits if `members.json` changes.

Reply:
I was thinking, we have an infinite loop, if we launch this at each push we have a risk of infinite loop.
Let's just transform the triggering into this this :
```
on:
  push:
    branches:
      - 'main'
    paths-ignore:
      - 'ecosystem/resources/members.json'
  workflow_dispatch:
```

Reply:
Nice catch! I actually think we can be more specific, and only trigger if one of the toml files changes (see [76f5ef5](https://github.com/qiskit-community/ecosystem/pull/426/commits/76f5ef56298700e4d5ec52476b4e4dd4bb905d07)).